Gruntfile.js

```diff
@@ -1,223 +1,225 @@
+          htmlmin: {
+            collapseBooleanAttributes:      true,
+            collapseWhitespace:             true,
+            removeAttributeQuotes:          true,
+            removeComments:                 false,
+            removeEmptyAttributes:          true,
+            removeRedundantAttributes:      true,
+            removeScriptTypeAttributes:     true,
+            removeStyleLinkTypeAttributes:  true
+          }
+        },
+        'patternfly.form': {
+          cwd: 'src/',
+          src: ['form/**/*.html'],
+          dest: 'templates/form.js'
+        },
+        'patternfly.notification': {
+          cwd: 'src/',
+          src: ['notification/**/*.html'],
+          dest: 'templates/notification.js'
+        },
+        'patternfly.card': {
+          cwd: 'src/',
+          src: ['card/**/*.html'],
+          dest: 'templates/card.js'
+        },
+        'patternfly.charts': {
+          cwd: 'src/',
+          src: ['charts/**/*.html'],
+          dest: 'templates/charts.js'
+        }
+      },
+      uglify: {
+        options: {
+          mangle: false
+        },
+        build: {
+          files: {},
+          src: 'dist/angular-patternfly.js',
+          dest: 'dist/angular-patternfly.min.js'
+        }
+      },
+      watch: {
+        main: {
+          files: ['Gruntfile.js'],
+          tasks: ['jshint']
+        },
+        js: {
+          files: ['Gruntfile.js', 'src/**/*.js'],
+          tasks: ['build']
+        }
+      }
+    });
+
+    // You can specify which modules to build as arguments of the build task.
+    grunt.registerTask('build', 'Create bootstrap build files', function() {
+      var concatSrc = [];
+
+      if (this.args.length) {
+        this.args.forEach(function(file) {
+          if (grunt.file.exists('./src/' + file)) {
+            grunt.log.ok('Adding ' + file + ' to the build queue.');
+            concatSrc.push('src/' + file + '/*.js');
+          } else {
+            grunt.fail.warn('Unable to build module \'' + file + '\'. The module doesn\'t exist.');
+          }
+        });
+
+      } else {
+        concatSrc = 'src/**/*.js';
+      }
+
+      grunt.task.run(['clean', 'lint', 'test', 'ngtemplates', 'concat', 'uglify:build', 'ngdocs', 'copy', 'clean:templates']);
+    });
+
+    grunt.registerTask('release', function (bump) {
+      // If no 'bump' argument was provided, set it to 'patch'
+      if (arguments.length === 0) {
+        bump = 'patch';
+      }
+
+      // Build first, bump the appropriate version piece, build the dist folder, and commit it all up
+      grunt.task.run('build', 'bump-only:' + bump, 'bump-commit');
+    });
+
+    grunt.registerTask('default', ['build']);
+    grunt.registerTask('ngdocs:view', ['build', 'connect:docs', 'watch']);
+    grunt.registerTask('lint', ['jshint', 'htmlhint']);
+    grunt.registerTask('test', ['karma']);
+    grunt.registerTask('help', ['availabletasks']);
+
+  }
+
+  init({});
+
+};
```

Comment:
Need to fix your editor config, it seems like there is some issue with whitespace changes here.